website/src/main.rs

```diff
@@ -0,0 +1,42 @@
+use stylist::{global_style, yew::styled_component};
+use yew::{html, Html};
+
+#[styled_component(Root)]
+fn root() -> Html {
+    global_style!(
+        r#"
```

Comment:
I think the strings are actually not necessary. The global_style! and css! macros understand CSS as a DSL; Strings are usually used here for the same reasons as format!(), because it allows variable substitution.